hw/ip/entropy_src/doc/_index.md

```diff
@@ -10,82 +10,95 @@ This module conforms to the [Comportable guideline for peripheral functionality.
 
 ## Features
 
-- This revision supports a LFSR (linear feedback shift register) digital source, and provides an interface to an external rng (random noise generator) source.
-The LFSR is used as a digital, pseudo-random type of entropy source, while the rng external source is a true entropy source.
+- This revision provides an interface to an external RNG (random noise generator) source, and a LFSR (linear feedback shift register) digital source. 
+The LFSR is used as a digital, pseudo-random type of entropy source, while the RNG external source is a digital noise source.
 A noise source and its relation to an entropy source are defined by [SP 800-90B](https://csrc.nist.gov/publications/detail/sp/800-90b/final).
-- A set of registers is provided for firmware to manage and obtain entropy bits.
-- The set of registers is designed such that firmware can select between a digital or an rng entropy source.
+- A set of registers is provided for firmware to obtain entropy bits.
+- The set of registers is designed such that firmware can select between an LFSR or an RNG digital noise source.
 - Interrupts are supported:
   - Entropy bits are available for firmware consumption.
-  - The external rng entropy source is available.
-  - The internal health tests have detected a repetitive count test fail.
-  - The internal health tests have detected an adaptive proportion test fail.
+  - The internal health tests have detected a test fail.
   - An internal FIFO error has occurred.
 - Two health checks that are defined by SP 800-90B are performed by this revision:  Repetition Count and Adaptive Proportion tests.
-- No alerts are supported by this revision.
+- Two additional health are performed by this revision:  Bucket and Markov tests.
+- A health fail alert is supported by this revision.
 
 
 ## Description
 
 This IP block provides an entropy source that is part of a larger solution to generate true random numbers (a TRNG mechanism).
 The general solution for TRNG relies on either hardware or firmware to implement this function, depending which functions are enabled by firmware or e-fuses.
-For purposes of reference in this document, the NIST term "noise" will be synonymous with the term "entropy".
-Likewise, no conditioning function will be performed in this block.
+A conditioning function will be performed in this block.
 For the generation of entropy, this block provides the entropy bits needed for the entropy source solution as required by the RNG mechanism.
 
 At a high-level, this ENTROPY_SRC block, when enabled, will continuously collect entropy bits from the entropy source into a FIFO that can be read from the TLUL bus, or sent out through a hardware interface.
-The rng source will generate a relative slow set of bit streams, in the tens of kilohertz range.
+The RNG source will generate a relatively slow set of bit streams, in the tens of kilohertz range.
```

Comment:
Honestly don't know - should this be "kilohertz" or "kilobits per second" (i.e. do you think of it as a frequency or an information transmission rate)?

Reply:
It's a good point because if you think about it, is actually 200 kbps.  Changed to "with a total bandwidth on the order of 100 kbps.